SU-HSDO/suhumsci
# SHS-5184: Implementation: Add active trail indicator to main nav

# READY FOR REVIEW

## Summary
Add missing active trail indicator to main nav

## Urgency
'low'

## Steps to Test
1. Navigate to a handful of pages to make sure the active trail is indicated as expected
2. Test on both Colorful and Traditional themes
3. Test with a variety of palettes
4. Test in all resolutions

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
